packages/platforms/accton/x86-64/as7726-32x/onlp/builds/x86_64_accton_as7726_32x/module/src/thermali.c

```diff
@@ -267,16 +268,32 @@ onlp_thermali_info_get(onlp_oid_t id, onlp_thermal_info_t* info)
     	case THERMAL_1_ON_PSU1:
     	case THERMAL_1_ON_PSU2:
 			format = PSU_THERMAL_PATH_FORMAT;
+
+            psu_id = ( tid == THERMAL_1_ON_PSU1 ) ? PSU1_ID : PSU2_ID;
+            /* Get power good status */
+            if (psu_status_info_get(psu_id, "psu_power_good", &val) != ONLP_STATUS_OK) {
+                AIM_LOG_ERROR("Unable to read PSU(%d) node(psu_power_good)\r\n", psu_id);
```

Comment:
There is already a log in the API psu_status_info_get(), do we still need to create another similar log?
AIM_LOG_ERROR("Unable to read status from file(%s)\r\n", path);

Reply:
Yes, this log is duplicated elsewhere and needs to be deleted here. I'll check all the modication.,